### ISSUE TITLE
Midi Note On/Off does not reach internal Carla-Rack inside Carla-Patchbay

### ISSUE THREAD

**Issue by diovudau** [falkTX/Carla#745]
```
git compiled 2018-09-15 23:00 CET
Using Carla version 1.9.11 (2.0-RC1)
  Python version: 3.7.0
  Qt version:     5.11.1
  PyQt version:   5.11.2
  Binary dir:     /usr/lib/carla
  Resources dir:  /usr/share/carla/resources
```

Steps to reproduce:

- Start carla-patchbay standalone from commandline
- Add carla-rack internal client to the patchbay
- Connect Midi Input:events-out to Carla-Rack:events-in
- Connect external midi source to Carla:events-in. for example jack-keyboard.
- Confirm in Carla-Rack "Edit" context menu that all MIDI checkboxes are checked (except CC)
- Open Carla-Rack, add plugin eg. Sherlock Inspector and open its GUI
- Create midi notes 0x80 or 0x90 from external source. They do not show up in sherlock (or any other plugin). Program changes are ignored as well.
- Other messages like Channel Pressure or Pitchbend  will work. 

Under certain conditions note on and off gets through. Try in jack keyboard to press down the mouse button and move rapidly left and right on the virtual keyboard. this should produce some note messages.

**Comment by falkTX** [comment]
can confirm, thanks for the report.
one final issue for 2.0-rc1 then.
will try to get it fixed tomorrow.

**Comment by falkTX** [comment]
Fixed in a468359963e75b1a09330770d67a0c8831c6d383
I added some changes in 31eb7760b0db3b22c0057a30f12399744cbb3e33 so we can catch whenever something like this happens again.
Thanks!
